src/pages/students/index.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import MyStudents from "@/components/case_manager/MyStudents";
+import MyStudents from "../../components/case_manager/MyStudents";
```

Comment:
Any specific reason for this change?

Reply:
Nope, I think this was an accidental change. Will revert.